src/screens/HyperSwitch/PaymentLogs/PaymentLogsUtils.res

```diff
@@ -10,3 +10,26 @@ let nameToURLMapper = (urlName, ~payment_id, ~merchant_id="", ()) => {
   | _ => urlName
   }
 }
+
+let filteredKeys = [
+  "value",
+  "merchant_id",
+  "created_at_precise",
+  "component",
+  "platform",
+  "version",
+]
+
+let sortByCreatedAt = (log1: Js.Json.t, log2: Js.Json.t) => {
+  open LogicUtils
+  let getKey = dict => dict->getDictFromJsonObject->getString("created_at", "")->Js.Date.fromString
+  let keyA = log1->getKey
+  let keyB = log2->getKey
+  if keyA < keyB {
```

Comment:
`LogicUtils.compareLogic() `

can be called directly need to pass KeyA and keyB